src/extension/ddappsec.c

```diff
@@ -308,8 +308,16 @@ int dd_appsec_rshutdown()
 {
     dd_conn *conn = dd_helper_mgr_cur_conn();
     if (conn) {
-        // currently does nothing
-        UNUSED(dd_request_shutdown(conn));
+        int res = dd_request_shutdown(conn);
+        if (res == dd_network) {
+            mlog_g(dd_log_info,
```

Comment:
Should this be error instead of info?

Reply:
There's other error logs before getting here so these are fine as info.